common/lazy-pages/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "gear-common-lazy-pages"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2018"
+license = "GPL-3.0"
+
+[dependencies]
+derive_more = "0.99.17"
+log = "0.4.17"
```

Comment:
disable default features and append std with std feature

Reply:
Done